setup.py

```diff
@@ -4,7 +4,7 @@
 from setuptools import setup
 
 version = '1.0.dev0'
-description = 'Portlets para Portal PadrÄĹo'
+description = 'Portlets para Portal Padrao'
```

Comment:
zest.releaser changed this line in commit 80c602f765ef25b0b77286e4787f343ae50b3085. zest.releaser is [trying very hard](https://github.com/zestsoftware/zest.releaser/blob/6.12.1/zest/releaser/utils.py#L89) to read the changelog and `setup.py` and only write back a change to the version number without touching other lines. `utf-8` should not be a problem, but in practice sometimes it is. Maybe it looks like `utf-8` but it isn't. Hard to say. Occasionally I have had to change a text to ascii to avoid problems like this. :-/